load_collection.json

```
@@ -295,7 +295,7 @@
         },
         {
             "rel": "about",
-            "href": "http://www.epsg-registry.org",
+            "href": "https://spatialreference.org/ref/epsg",
             "title": "Official EPSG code registry"
```

It's not the offibial EPSG code registry, unfortunately:

```suggestion
            "title": "EPSG code registry"
```